### PR TITLE
Replace saiUI with bslib

### PR DESCRIPTION
**Pull request by Aeilert**
Hi @ohjakobsen,

Here is an initial attempt at solving #2. Please note the following: 

* The functionality to hide the "Analysis" and "Malmquist" tabs on application start up doesn't work, even with the discussed workaround in `pioneer.js`. Any suggestion you have on this is welcome. 
* There are some minor changes in the layout, including the file-input box and colouring. We might want to tweak this. 
* Great if you could go over the overall functionality as you are more familiar with the app, and there might be something that I have missed. 
* As a simple solution `dropdown_button()` in utils.R is just a copy-paste from `saiUI::bs4dropdown()`. Let me know if you think we should do this differently. 

Feel free to implement any changes directly if you prefer.

### CODE REVIEW

**Comment by ohjakobsen** [comment]
The app fails because of dependency on the function `singleLayout()` from saiUI. `singleLayout()` can be replaced with:
```r
tags$div(class = 'container mt-1', tags$div(class = 'row', tags$div(class = 'col-12', ...)))
```
Also note that the function `card()` was added to bslib 0.4.2. This should be reflected in `DESCRIPTION`.

**Comment by Aeilert** [comment]
> The app fails because of dependency on the function `singleLayout()` from saiUI. `singleLayout()` can be replaced with:
> 
> ```r
> tags$div(class = 'container mt-1', tags$div(class = 'row', tags$div(class = 'col-12', ...)))
> ```
> 
> Also note that the function `card()` was added to bslib 0.4.2. This should be reflected in `DESCRIPTION`.

This should now be fixed.

**Comment by ohjakobsen** [comment]
I've added a fix for the JS issue to support hiding/showing tabs depending on the type of analysis selected. There is still a bug in the Malmquist tab that needs to be solved. I'm going to keep the review open until I've found a fix.

File: inst/app/global.R [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Load packages
-require(saiUI)
+library(shiny)
+library(bslib)
+source('utils.R')
```
Comment (ohjakobsen) [inline inst/app/global.R:4]: Move `utils.R` to R directory inside `./inst/app/`. Files inside this structure is automatically sourced with shiny 1.5.0 and above

**Review by ohjakobsen — CHANGES_REQUESTED** [review]
Move `utils.R` to proper location. I've added several commits to fix bugs with this PR. All changes should be verified.

**Comment by Aeilert** [comment]
@ohjakobsen: I moved the utils.R file inside inst/app/R and have verified that the  functionality to hide the "Analysis" and "Malmquist" tabs on start-up works, as well as swapping between them based on other user selections.

**Review by ohjakobsen — APPROVED** [review]

**Pull request merged**